### ISSUE TITLE
Change the link given for members in the "Currently contributing" column

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#2917]
Change the link given for members in the "Currently contributing" column on https://tatoeba.org/en/contributions/latest

FROM:
https://tatoeba.org/en/contributions/of_user/doemaar14
TO:
https://tatoeba.org/en/activities/translate_sentences_of/doemaar14

This would be more useful for members.  It's definitely what I'm looking for, so requires several clicks to get there the way things are now.

### https://tatoeba.org/en/contributions/latest

<img width="303" alt="Screen Shot 2022-04-05 at 8 38 54" src="https://user-images.githubusercontent.com/7934273/161649830-7382603d-917e-4f5f-b877-17775be749be.png">

### https://tatoeba.org/en/contributions/of_user/doemaar14
<img width="702" alt="Screen Shot 2022-04-05 at 8 41 50" src="https://user-images.githubusercontent.com/7934273/161649976-a34870be-ebff-466d-b961-00a8f12ef9b1.png">

### https://tatoeba.org/en/activities/translate_sentences_of/doemaar14
<img width="659" alt="Screen Shot 2022-04-05 at 8 42 34" src="https://user-images.githubusercontent.com/7934273/161649986-0f7b1491-3bbf-4317-9a8e-526d0e538596.png">

**Comment by kumakyoo42** [comment]
I agree that a more prominent link to "translate sentences of" would be a nice idea. But I doubt that the current contributions column is the right place for doing so, because this list is about contributions and I expect to see something about their contributions when clicking the link.
